packages/block/src/helpers.ts

```diff
@@ -127,16 +132,28 @@ export const fakeExponential = (factor: bigint, numerator: bigint, denominator:
   return output / denominator
 }
 
+export const getBlobGasSchedule = (common: Common): BpoSchedule => {
+  if (common.gteHardfork(Hardfork.Bpo1)) {
+    return getBpoScheduleFromHardfork(common.hardfork())
+  }
+  return {
+    targetBlobGasPerBlock: common.param('targetBlobGasPerBlock'),
+    maxBlobGasPerBlock: common.param('maxBlobGasPerBlock'),
+    blobGasPriceUpdateFraction: common.param('blobGasPriceUpdateFraction'),
+  }
+}
+
 /**
  * Returns the blob gas price depending upon the `excessBlobGas` value
  * @param excessBlobGas
  * @param common
  */
 export const computeBlobGasPrice = (excessBlobGas: bigint, common: Common) => {
+  const schedule = getBlobGasSchedule(common)
   return fakeExponential(
     common.param('minBlobGas'),
     excessBlobGas,
-    common.param('blobGasPriceUpdateFraction'),
+    schedule.blobGasPriceUpdateFraction,
```

Comment:
This should actually continue to work in the old way, I will update here. If not, then we need to rework this HF-based params section in Common to some extend, cause then it's too brittle. The Common part *should* work under these circumstances, and e.g. be clever enough to choose the correct value if there is a param from an EIP in an older HF + a new direct param from a HF.